config/action.d/shorewall.conf

```diff
@@ -50,10 +52,33 @@ actionban = shorewall <blocktype> <ip>
 #
 actionunban = shorewall allow <ip>
 
-[Init]
-
 # Option:  blocktype
 # Note:    This is what the action does with rules.
 #          See man page of shorewall for options that include drop, logdrop, reject, or logreject
 # Values:  STRING
 blocktype = reject
+
+[Init?family=inet6]
+
```

Comment:
The test cases will fail, as long as `actionban` and `actionunban` belong to section definition.

So just create a new init parameter `family` (empty for ipv4 and `6` for ipv6):
```
[Definition]
actionban = shorewall<family> <blocktype> <ip>
actionunban = shorewall<family> allow <ip>

[Init]
family =

[Init?family=inet6]
family = 6
```